docs/API.md

```diff
@@ -480,31 +480,40 @@ A single glob or array of globs that indicate which files to watch for changes.
 #### opts
 Type: `Object`
 
-Options, that are passed to [`gaze`][gaze].
+Options, that are passed to [`chokidar`][chokidar].
 
 #### fn
 Type: `Function`
 
 An [async](#async-support) function to run when a file changes.
 
-`gulp.watch` returns an `EventEmitter` object which emits `change` events with
-the [gaze] `event`:
+`gulp.watch` returns an `FSWatcher` object that already listens to the `change`
```

Comment:
Suggested edit:

returns a wrapped [chokidar] `FSWatcher` object. If provided, the callback will be triggered upon any `add`, `change`, or `unlink` event. Listeners can also be set directly for any of [chokidar]'s events.